doc/api_blueprints/elektrad.apib

```diff
@@ -15,120 +15,156 @@ The API version is increased whenever breaking changes
+
++ Response 200
+   
++ Response 400 (application/json; charset=utf-8)
     + Attributes (Error)
-        + i18n (string)
-            A comprehensive list of possible errors:
-            - `PATH_NOT_FOUND` if the requested path does not exist in the key database
 
-### set configuration [PUT]
 
-+ Request (text/plain)
+## metakeys [/kdbMeta/{+path}]
 
-        hello world
+### create metakey [POST]
 
-+ Response 204
++ Attributes (Metakey)
 
 + Request (application/json)
-
-        "hello world"
+    + Parameters
+        + path: `user/hello` (string) - path to the elektra config
 
 + Response 204
 
-+ Response 400 (application/json)
++ Response 400 (application/json; charset=utf-8)
     + Attributes (Error)
-        + i18n (string)
-            A comprehensive list of possible errors:
-            - `APIKEY_MISSING` api key not specified
 
-+ Response 401 (application/json)
-    + Attributes (Error)
-        + i18n (string)
-            A comprehensive list of possible errors:
-            - `NEED_AUTHENTICATION` if no valid apikey has been supplied
+### delete metakey [DELETE]
 
-+ Response 404 (application/json)
-    + Attributes (Error)
-        + i18n (string)
-            A comprehensive list of possible errors:
-            - `PATH_NOT_FOUND` if the requested path does not exist in the key database
++ Attributes (Metakey)
+
++ Request (application/json)
+    + Parameters
+        + path: `user/hello` (string) - path to the elektra config
 
-+ Response 406 (application/json)
++ Response 204
+
++ Response 400 (application/json; charset=utf-8)
     + Attributes (Error)
-        + i18n (string)
-            A comprehensive list of possible errors:
-            - `REQUEST_UNSUPPORTED_CONTENT_TYPE` if the submitted Content-Type is not `application/json` or `text/plain`
 
-### delete configuration [DELETE]
+
+## Move operation [POST /kdbMv/{+from}]
+
+move a key (and all its subkeys) to a new path - works like `kdb mv -r`
+
++ Request (text/plain)
+    + Parameters
+        + from: `user/hello` (string) - path to the elektra config
+
+    + Body
+
+            user/toPath
 
 + Response 204
 
-+ Response 400 (application/json)
++ Response 400 (application/json; charset=utf-8)
     + Attributes (Error)
-        + i18n (string)
-            A comprehensive list of possible errors:
-            - `APIKEY_MISSING` api key not specified
 
-+ Response 401 (application/json)
-    + Attributes (Error)
-        + i18n (string)
-            A comprehensive list of possible errors:
-            - `NEED_AUTHENTICATION` if no valid apikey has been supplied
 
-+ Response 404 (application/json)
+## Copy operation [POST /kdbCp/{+from}]
+
+copy a key (and all its subkeys) to a new path - works like `kdb cp -r`
+
++ Request (text/plain)
+    + Parameters
+        + from: `user/hello` (string) - path to the elektra config
+
+    + Body
+
+            user/toPath
+
++ Response 204
+
++ Response 400 (application/json; charset=utf-8)
```

Comment:
In general there should be much more here so that dredd has something to do.

errors? different requests?

Alternatively (or additionally) some curl test cases would be nice. (E.g. in shell recorder syntax.)